mbuild/formats/lammpsdata.py

```diff
@@ -99,7 +116,10 @@ def write_lammpsdata(structure, filename, atom_style='full'):
                                                      round(dihedral.type.c4,3),
                                                      round(dihedral.type.c5,3),
                                                      round(dihedral.type.scee,1),
-                                                     round(dihedral.type.scnb,1)) for dihedral in structure.rb_torsions])))
+                                                     round(dihedral.type.scnb,1)
+                                                     ,str(dihedral.atom1.type), str(dihedral.atom2.type),
```

Comment:
inconsistent comma

Place the commas at the end of each line, for easier readability.